src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/OutboundEventsService.kt

```diff
@@ -19,3 +39,45 @@ class OutboundEventsService(
     outboundEventsPublisher.send(event)
   }
 }
+
+@Profile("local")
+@Service
+class LocalOutboundEventsService(
```

Comment:
Added this for running locally if we want to create/amend in the external API's simply bypassing the sending of events.  Easy enough to remove so happy to do so and replace with localstack.

Reply:
Looks good to me - we can enable if required, otherwise carry on without.